eth/tracers/tracer.go

```diff
@@ -312,10 +312,18 @@ type Tracer struct {
 	reason    error  // Textual reason for the interruption
 }
 
+// Context contains some contextual infos for a transaction execution that is not
+// available from within the EVM object.
+type Context struct {
+	BlockHash common.Hash // Hash of the block the tx is contained within (zero if dangling tx or call)
```

Comment:
I probably missed the memo, just wondering why you exported the fields?

Reply:
It's part of the API (`tracers.New`).